doc/api/v8.md

```diff
@@ -542,8 +542,248 @@ added: v8.0.0
 A subclass of [`Deserializer`][] corresponding to the format written by
 [`DefaultSerializer`][].
 
+## Promise hooks
+
+The `promise_hooks` interface can be used to track promise lifecycle events.
+To track _all_ async activity, see [`async_hooks`][] which internally uses this
+module to produce promise lifecycle events in addition to events for other
+async resources. For request context management, see [`AsyncLocalStorage`][].
+
+```mjs
+import { PromiseHooks } from 'v8';
```

Comment:
AFAIK the convention is to normally do `PromiseHooks` when it's a class (or something that you can `new`) and `promiseHooks` if it's just a namespace. Is there a particular reason that it's `PromiseHooks` here?

Reply:
Also we should probably update the mention of `promise_hooks ` above with whatever the namespace is going to be.

Reply:
No particular reason. I can change it. 👍